mantidproject/mantidimaging
# Adds initial reconstruction docs

Adds initial reconstruction section as part of the user guide, and some docs for the current available algorithms. Adds some brief explanation as well, because the actual ASTRA and TomoPy docs don't really say anything themselves..

- Also refactors the help calling code to make it more re-usable

To test:

- Check that operation pages still open properly
- Check that the reconstruction help button (in the reconstruct tab) opens to the correct URL - you will get a 404 as this PR needs to be published to see it.
- Have a read through the `reconstruction/algorithms.rst`

Fixes #663
